README.md

```diff
@@ -8,13 +8,14 @@
 
 ### Configuration
 The configuration file contains following properties:
-- `host` - string (required): host or ip address to your sisense workspace
-- `port` - string (optional): port where is running sisence (default 30845)
-- `username` - string (required): username for sisense
-- `#password` - string (required): password for sisense
-- `datamodel` - string (required): name of datamodel in sisense
+- `db`
+    - `host` - string (required): host or ip address to your sisense workspace
+    - `port` - string (optional): port where is running sisence (default 30845)
+    - `username` - string (required): username for sisense
+    - `#password` - string (required): password for sisense
+- `dbName` - string (required): name of datamodel in sisense
```

Comment:
Databaza zvykne byt vacsinou v `db.database`, ci nie?

Reply:
to je otázkou kde se to bude nastavovat
db.database je společný pro včechny rows a dbName si můžeš nastavit pokaždé třeba jiné.
a pokud máme jedny přihlašovací údaje do sisence a můžeš tam zakládat více datamodelů tak bych to nechal tady

Reply:
ok ... v tom pripade to mohlo asi zostat ako `datamodel`

Reply:
je to sjednoceno podle ostatních db writerů:
https://github.com/keboola/db-writer-mysql/blob/master/src/Configuration/MySQLConfigRowDefinition.php#L50
https://github.com/keboola/db-writer-mssql/blob/master/src/MSSQL/Configuration/MssqlConfigRowDefinition.php#L57